Exercise_4/interactive_commad_list

```diff
@@ -0,0 +1,33 @@
+## Be sure to install PyTorch and Python packages via
+# https://rc-docs.northeastern.edu/en/latest/using-discovery/workingwithgpu.html?highlight=pytorch#using-gpus-with-pytorch
+## For exercise 4 use:
+srun --partition=gpu --nodes=1 --ntasks=2 --gres=gpu:1 --pty --time=00:30:00 /bin/bash
+
+# Load modules and Python Environment
+conda deactivate
+module purge
+
+module load discovery
+module load cuda/11.7
+module load anaconda3/2022.05
+
+source activate pytorch_env
+
+# Go to the directory for Exercise_4
+cd $HOME/training-checkpointing
+cd Exercise_4
+
+# Train for a single epoch
+train_with_checkpoints.py --help
```

Comment:
This command doesn't work and doing

python train_with_checkpoints.py --help 

results in syntax error.